modules/ztp-install-openshift-pipelines.adoc

```diff
@@ -0,0 +1,94 @@
++
+[source,terminal]
+----
+$ curl -sLK https://raw.githubusercontent.com/rh-ecosystem-edge/ztp-pipeline-relocatable/main/pipelines/bootstrap.sh | bash -s -- ${KUBECONFIG}
+----
++
+This script:
+
+* Installs the `tkn` CLI. This tool manages {product-title} pipelines from a terminal.
+* Clones the link:https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable[ztp-pipeline-relocatable] pipeline repository.
+* Checks that the correct permissions are set on the hub cluster.
+* Deploys the {product-title} Pipelines Operator from the Operator Lifecycle Manager (OLM) catalog.
+* Deploys the ZTP pipeline and associated tasks.
+
+. Monitor the progress in the terminal window and in the web console UI.
+
+.. In the terminal window you are expected to see the following:
++
+[source,terminal]
+----
+>>>> Creating NS spoke-deployer and giving permissions to SA spoke-deployer
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+namespace/spoke-deployer configured
+serviceaccount/spoke-deployer configured
+clusterrolebinding.rbac.authorization.k8s.io/cluster-admin-0 configured
+
+>>>> Cloning Repository into your local folder
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+Cloning into 'ztp-pipeline-relocatable'...
+remote: Enumerating objects: 3824, done.
+remote: Counting objects: 100% (1581/1581), done.
+remote: Compressing objects: 100% (963/963), done.
+remote: Total 3824 (delta 963), reused 1163 (delta 589), pack-reused 2243
+Receiving objects: 100% (3824/3824), 702.12 KiB | 8.46 MiB/s, done.
+Resolving deltas: 100% (2182/2182), done.
+
+>>>> Deploying Openshift Pipelines
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+subscription.operators.coreos.com/openshift-pipelines-operator-rh unchanged
+>>>> Waiting for: Openshift Pipelines
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+>>>> Deploying ZTPFW Pipelines and tasks
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+pipeline.tekton.dev/deploy-ztp-hub configured
+pipeline.tekton.dev/deploy-ztp-spokes configured
+task.tekton.dev/common-pre-flight configured
+task.tekton.dev/hub-deploy-acm configured
+task.tekton.dev/hub-deploy-disconnected-registry configured
+task.tekton.dev/hub-deploy-httpd-server configured
+task.tekton.dev/hub-deploy-hub-config configured
+task.tekton.dev/hub-deploy-icsp-hub configured
+task.tekton.dev/hub-save-config configured
+task.tekton.dev/spoke-deploy-disconnected-registry-spokes configured
+task.tekton.dev/spoke-deploy-icsp-spokes-post configured
+task.tekton.dev/spoke-deploy-icsp-spokes-pre configured
+task.tekton.dev/spoke-deploy-metallb configured
+task.tekton.dev/spoke-deploy-ocs configured
+task.tekton.dev/spoke-deploy-spoke configured
+task.tekton.dev/spoke-deploy-workers configured
+task.tekton.dev/spoke-detach-cluster configured
+task.tekton.dev/spoke-restore-hub-config configured
+----
+
+.. Log in to the {product-title} web console.
+
+... Navigate to *Home* -> *Projects* and select the Project *openshift-operators*.
+
+... Select *Workloads*
```

Comment:
This seemed to end abruptly. Should there be another step for them to check something in the console?